packages/vite-plugin-cloudflare/playground/vitest-setup.ts

```diff
@@ -181,7 +181,11 @@ beforeAll(async (s) => {
 }, 15_000);
 
 beforeEach(async () => {
-	await page.goto(viteTestUrl);
+	try {
+		await page.goto(viteTestUrl);
+	} catch (error) {
+		// Expected to error for middleware-mode test as `viteTestUrl` will not be valid
```

Comment:
Could we make `viteTestUrl` valid somehow?

Reply:
I found a way to do this by exporting `viteTestUrl` from the custom serve file and then capturing that. Much nicer now.

[f469040](https://github.com/cloudflare/workers-sdk/pull/9108/commits/f469040092e02c0e1fc93cacc1b526a17c46eb64)